mars/services/lifecycle/supervisor/tracker.py

```diff
@@ -235,8 +240,18 @@ async def decref_tileables(
             list(decref_chunk_key_to_counts),
             counts=list(decref_chunk_key_to_counts.values()),
         )
-        # make _remove_chunks release actor lock
-        yield self._remove_chunks(to_remove_chunk_keys)
+        to_remove_tileable_keys = await asyncio.to_thread(
```

Comment:
Will there be many items in `tileable_keys`? If not, `asyncio.to_thread` is unnecessary

Reply:
Tileable keys are less than chunk keys, but I am not sure how many keys are here. I think `asyncio.to_thread` may be slower than iterate in asyncio loop directly for small data sets, but it can avoid blocking loop issues which is hard to findout.